articles/azure-functions/durable-functions-overview.md

```diff
@@ -57,6 +57,8 @@ public static async Task<object> Run(DurableOrchestrationContext ctx)
     }
 }
 ```
+> [!NOTE]
+> Above example is for C# script as indicated. There are subtle differences while writing a precompiled durable function in C# vs the C# script sample shown above. A C# precompiled function would require durable parameters to be decorated with respective attributes. An example is ```[OrchestrationClient]``` attribute for ```DurableOrchestrationClient``` parameter. If the parameters are not properly decorated, the runtime would not be able to inject the variables to the function and would give error. Please visit [sample](https://github.com/Azure/azure-functions-durable-extension/blob/master/samples/precompiled/HttpStart.cs) for more examples.
```

Comment:
- Instead of "Above example", use "The previous example" ("above" doesn't make sense for folks who use screen readers)
- I would change the link to (https://github.com/Azure/azure-functions-durable-extension/blob/master/samples, rather than pointing to HttpStart.cs
- The example here doesn't show `DurableOrchestrationClient`, so I would instead use `[OrchestrationTrigger]` and `DurableOrchestrationContext` as your examples.
- Optional, consider removing the sentence starting with "If the parameters are not properly decorated...". I'm just worried it might be too much text. The fact that you mentioned the difference in requirements for precompiled C# seems sufficient to me - but again...optional.